README.md

```diff
@@ -220,121 +94,6 @@ Tutorials
     * Basic cookiecutter template for cmd2 application : https://github.com/jayrod/cookiecutter-python-cmd2
     * Advanced cookiecutter template with external plugin support : https://github.com/jayrod/cookiecutter-python-cmd2-ext-plug
 
-Example Application
```

Comment:
I like showing at least a basic example application in the README.  I'm game for simplifying it.  But I want someone to quickly see what a "Hello World" `cmd2` application might look like on the "front page".